scripts/tests/cirque_tests.sh

```diff
@@ -72,6 +72,7 @@ function cirquetest_bootstrap() {
     cd "$REPO_DIR"/third_party/cirque/repo
     pip3 install pycodestyle==2.5.0 wheel
     make NO_GRPC=1 install -j
+    "$REPO_DIR"/integrations/docker/images/chip-cirque-device-base/build.sh
     ./dependency_modules.sh
```

Comment:
we need to run dedicated dependency_moduels.sh in chip instead of using reference one from cirque.

Reply:
per offline chat, song will do it in another pr via using commit id from openthread submodule